persistence/src/commonMain/kotlin/com/wire/kalium/persistence/client/AuthTokenStorage.kt

```diff
@@ -28,7 +28,7 @@ import kotlinx.serialization.Serializable
 data class AuthTokenEntity(
     @SerialName("user_id") val userId: UserIDEntity,
     @SerialName("access_token") val accessToken: String,
-    @SerialName("refresh_token") val refreshToken: String,
+    @SerialName("refresh_token") val refreshToken: String?,
```

Comment:
question: why is the refresh token nuable here ?

Reply:
We can throw an Exception, instead if refreshToken is null and not propagate the nullabled field

Reply:
I think it would be cleaner if we throw, then.